query-connector/README.md

```diff
@@ -77,7 +77,36 @@ The Query Connector will eventually require other inputs from other DIBBs servic
 
 ### Developer Documentation
 
-Can be found in [api-documentation.md](api-documentation.md).
+A Postman collection demonstrating use of the API can be found [here](https://github.com/CDCgov/dibbs-query-connector/blob/main/query-connector/src/app/assets/DIBBs_Query_Connector_API.postman_collection.json).
+
+### Query Connector Data for Query Building
+
+When initializing, the Query Connector makes the value sets associated with 200+ reportable conditions available to users tasked with building queries for their jurisdiction. To group value sets by condition and to group the conditions by type, the Query Connector obtains and organizes data from the eRSD and the VSAC in the following way:
+
+1. The Query Connector retrieves the 200+ reportable conditions from the eRSD as well as the value sets associated value set IDs.
```

Comment:
Nit: there's a typo / redundancy with "value sets associated value set IDs" (I think missing a conjunction), but I might modify this to something like: "From the eRSD, the Query Connector retrieves 200+ reportable conditions as well as a collection of clinical value sets and their unique IDs, grouping them by clinical category." This cleans up the typo language and also references how we determine labs vs condition vs medications classification.